scripts/lib.sh

```diff
@@ -149,7 +149,7 @@ setup-git() {
         exit 1
     )
     if ! require pre-commit; then
-        pip install -r requirements-dev-only-frozen.txt
+        pip install -r requirements-dev.txt
```

Comment:
IIUC, the _generated_ `requirements-dev-only-frozen.txt` is now replacing the original non-generated file.

Reply:
yeah, you're commenting on an older commit which included a merge error, `requirements-dev-only-frozen.txt` is intended